### PR TITLE
Add udev config for KVM to k8s install DaemonSet

### PR DESCRIPTION
**Pull request by johnmuth81**
Fixes: #1374

### CODE REVIEW

**Review by sv641 — APPROVED** [review]

File: cloud/k8s/deploy/k8s-deploy.sh [hunk 1]
```diff
@@ -67,8 +67,12 @@ function configure_cri_runtime() {
 		configure_containerd
 		;;
 	esac
-	[ -c /host-devices/kvm ] && chmod 666 /host-devices/kvm
-	[ -c /host-devices/kkm ] && chmod 666 /host-devices/kkm
+
+        [ -c /host-dev/kvm ] && chmod 666 /host-dev/kvm
```
Comment (gnode1) [inline cloud/k8s/deploy/k8s-deploy.sh:71]: This should be superfluous, the `udevadm trigger` should trigger the appropriate event and udev itself should change the permissions.

File: cloud/k8s/deploy/k8s-deploy.sh [hunk 2]
```diff
@@ -67,8 +67,12 @@ function configure_cri_runtime() {
...
+        mkdir -p /etc/udev/rules.d
+        echo 'KERNEL=="kvm", GROUP="kvm", MODE="0666"' | tee /etc/udev/rules.d/99-perm.rules
+        udevadm control --reload-rules && udevadm trigger
```
Comment (gnode1) [inline cloud/k8s/deploy/k8s-deploy.sh:74]: This runs in the container, I wonder if does what we need to happen. IOW which udev does it kick?

Comment (johnmuth81) [reply cloud/k8s/deploy/k8s-deploy.sh:74]: The YAMP file maps the node's `/var/run/dbus` and `/run/system` into the DaemonSet container. The reason for this is so things like `systemctl` and `udevadm` run from the DamonSet end up talking to the Node's daemons.

File: cloud/k8s/deploy/k8s-deploy.sh (hunk 1 above)
Comment (johnmuth81) [reply cloud/k8s/deploy/k8s-deploy.sh:71]: What about if udev isn't running at all? Isn't that possible?

File: cloud/k8s/deploy/k8s-deploy.sh (hunk 2 above)
Comment (gnode1) [reply cloud/k8s/deploy/k8s-deploy.sh:74]: `udevadm control --reload-rules` will reload this container's udev, not the host's, which means the host's one will not know of the change

Comment (johnmuth81) [reply cloud/k8s/deploy/k8s-deploy.sh:74]: Latest commit maps udev socket so udevadm talks to Node daemon.

**Review by gnode1 — APPROVED** [review]

**Pull request merged**